airbyte-cdk/python/connector_builder/README.md

```diff
@@ -0,0 +1,32 @@
+# Connector Builder Backend
```

Comment:
should the readme be in the `source_declarative_manifest` module since that's where the main is?

Reply:
That has a separate README, which is to do with running the Connector Builder manifests. Based on this comment I added a line in there to indicate that it handles Connector Builder requests too, but feel like it makes sense for this to have its own README.